app/lib/actions/katello/repository/clone_to_version.rb

```diff
@@ -25,6 +25,9 @@ def plan(repositories, content_view_version, incremental = false)
                   plan_action(Repository::CloneYumContent, repository, new_repository, filters, !incremental,
                               :generate_metadata => !incremental, :index_content => !incremental, :simple_clone => incremental)
                 end
+              elsif new_repository.deb?
+                plan_action(Repository::CloneDebContent, repository, new_repository, filters, !incremental,
+                            :generate_metadata => !incremental, :index_content => !incremental, :simple_clone => incremental)
```

Comment:
I actually broke this!  notice that the repository variable isn't defined, I fixed it here:  https://github.com/Katello/katello/pull/7121

Reply:
I guess, we need to wait on this one then?